packages/origin-backend/src/pods/configuration/configuration.entity.ts

```diff
@@ -0,0 +1,30 @@
+import { Entity, PrimaryColumn, BaseEntity, Column, Check } from 'typeorm';
+import { ExternalDeviceIdType, IOriginConfiguration } from '@energyweb/origin-backend-core';
+
+@Entity()
+@Check(`id = 1`)
+export class Configuration extends BaseEntity implements IOriginConfiguration {
+    @PrimaryColumn({type: 'int', default: () => `1`, nullable: false})
+    id: 1;
+
+    @Column('varchar', { nullable: true })
+    countryName: string;
+
+    @Column('simple-array', { nullable: true })
+    currencies: string[];
+
+    @Column('varchar', { nullable: true })
+    regions: string;
```

Comment:
It's ok for now, but we should come up with a type that will represent regions and theirs matching capabilities.

Reply:
Thanks, done.